services/wallet/bridge/cbridge.go

```diff
@@ -289,13 +289,25 @@ func (s *CBridge) Send(sendArgs *TransactionBridge, verifiedAccount *account.Sel
 			500,
 		)
 	}
+}
+
+func (s *CBridge) Send(sendArgs *TransactionBridge, verifiedAccount *account.SelectedExtKey) (types.Hash, error) {
+	tx, err := s.send(sendArgs, getSigner(sendArgs.ChainID, sendArgs.CbridgeTx.From, verifiedAccount))
 	if err != nil {
 		return types.HexToHash(""), err
 	}
 
 	return types.Hash(tx.Hash()), nil
 }
 
+func (s *CBridge) SendWithSignature(chainID uint64, tx *ethTypes.Transaction, signature []byte) (types.Hash, error) {
+	return s.transactor.SendBuiltTransactionWithSignature(chainID, tx, signature)
```

Comment:
Do we need a SendWithSignature being copied over all bridges and doing the same, this could simply be trigger from transfer/transaction

As we don't have custom logic per bridge?

(Comment here is about the interface so valid for all bridges)

Reply:
Yes, we can, I didn't know that all of them would do the same thing at the beginning and added that to the interface, now when I see they do exactly the same thing, it can be moved.